resolwe_bio/docker_images/base/Dockerfile.ubuntu-14.04

```diff
@@ -0,0 +1,65 @@
+FROM docker.io/resolwe/base:ubuntu-14.04
+
+MAINTAINER Genialis <dev-team@genialis.com>
+
+# Add common build helpers.
+ADD common.sh /var/cache/build/packages-manual/common.sh
```

Comment:
Because this is something only needed during build and could be removed later (although we currently don't remove it). Should we put this somewhere else? I can create something like `/build`.